src/App.jsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react/jsx-no-constructed-context-values */
+import { useState } from 'react';
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+
+import './App.css';
+import CourseInfo from './components/CourseInfo/CourseInfo';
+import Courses from './components/Courses/Courses';
+import CreateCourse from './components/CreateCourse/CreateCourse';
+import Header from './components/Header/Header';
+import Login from './components/Login/Login';
+import Registration from './components/Registration/Registration';
+import Auth from './auth/Auth';
+import LoginAuth from './auth/LoginAuth';
+import { AuthContext } from './auth/AuthContext';
+import checkIfUserLogged from './helpers/checkIfUserLogged';
+
+function App() {
+  const [loginStatus, setLoginStatus] = useState(checkIfUserLogged());
```

Comment:
Well, it should not call too often now, but I still consider this as a not very good practice